R/input.R

```diff
@@ -179,11 +179,18 @@ textInput <- function(inputId, label, value = "", width = NULL,
 #' }
 #'
 #' @export
-numeric_input <- function(input_id, label, value, min = NA, max = NA, step = NA,
+numeric_input <- function(input_id, label, value = NULL, min = NA, max = NA, step = NA,
                            type = NULL, icon = NULL, placeholder = NULL, ...) {
-  if (!is.numeric(value) & !grepl("^\\d*(\\.\\d*|)$", value)) stop("Non-numeric input detected")
+  if (!(is.null(placeholder) || is.character(placeholder))) {
+    stop ("'placeholder' should be NULL or character")
+  }
+  if (is.null(value) & is.null(placeholder)) stop ("either 'value' or 'placeholder' should be defined")
```

Comment:
This requirement should be also included in the function documentation.

Reply:
Done